src/graphql.ts

```diff
@@ -48,44 +59,50 @@ export function InterfaceTypeDefinitionBuilder(useInterfaceTypes: boolean | unde
 export function topologicalSortAST(schema: GraphQLSchema, ast: DocumentNode): DocumentNode {
   const dependencyGraph = new Graph();
   const targetKinds = [
-    'ObjectTypeDefinition',
-    'InputObjectTypeDefinition',
-    'EnumTypeDefinition',
-    'UnionTypeDefinition',
-    'ScalarTypeDefinition',
+    Kind.OBJECT_TYPE_DEFINITION,
+    Kind.INPUT_OBJECT_TYPE_DEFINITION,
+    Kind.INTERFACE_TYPE_DEFINITION,
+    Kind.SCALAR_TYPE_DEFINITION,
+    Kind.ENUM_TYPE_DEFINITION,
+    Kind.UNION_TYPE_DEFINITION,
   ];
 
   visit<DocumentNode>(ast, {
     enter: (node) => {
       switch (node.kind) {
-        case 'ObjectTypeDefinition':
-        case 'InputObjectTypeDefinition': {
+        case Kind.OBJECT_TYPE_DEFINITION:
+        case Kind.INPUT_OBJECT_TYPE_DEFINITION:
+        case Kind.INTERFACE_TYPE_DEFINITION: {
```

Comment:
This is a new case, so please add a test to confirm that the interface is also sorted